packages/ember-views/lib/views/states/destroying.js

```diff
@@ -9,6 +10,12 @@ import _default from './default';
 const destroying = Object.create(_default);
 
 assign(destroying, {
+  $(view, sel) {
```

Comment:
@krisselden Thanks for review! View mixin seems to be the most appropriate place for that method. PR updated.